packages/react-core/src/demos/Button.md

```diff
@@ -5,9 +5,9 @@ section: components
 
 import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
 
-## Demos
+## Progress button
 
-### Progress button
+The following demo shows the intended flow for a button that visually indicates progress. This example demonstrates a login process, which updates the button label based on the `loginState`.
```

Comment:
Just to clarify the purpose of the demo, we could possibly add in some verbiage about how the input fields can't be edited and only the button can be interacted with or something along those lines. @mcarrano what do you think?